frontend/ui-pablo/app/components/Organisms/liquidity/RemoveForm/index.tsx

```diff
@@ -119,10 +113,14 @@ export const RemoveLiquidityForm: React.FC<BoxProps> = ({ ...rest }) => {
   useEffect(() => {
     confirmed && dispatch(closeConfirmingModal());
     !confirmed && dispatch(setMessage({}));
+    // Dispatch doesn't need to be added to the dependency list. 
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [confirmed]);
 
   useEffect(() => {
     dispatch(setMessage({}));
+    // Dispatch doesn't need to be added to the dependency list. 
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
Why wouldn't we add dispatch to the list?

Reply:
dispatch is meant to remain the same thoughout, same as `setters`

Reply:
Yes, but in that case it should be ok to put it in the list, shouldn't it? Anyway, you can resolve this comment both ways. :))